build/charts/theia/provisioning/datasources/migrators/000005_0-6-0.down.sql

```diff
@@ -190,3 +190,13 @@ INSERT INTO ".inner.flows_policy_view_local" SELECT * FROM policy_view_table_loc
 DROP TABLE pod_view_table_local;
 DROP TABLE node_view_table_local;
 DROP TABLE policy_view_table_local;
+
+--Alter table to drop new columns
+ALTER TABLE flows
```

Comment:
Will the following statement work?
```sql
ALTER TABLE flows DROP egressName String, DROP egressIP String;
ALTER TABLE flows_local DROP egressName String, DROP egressIP String;
```

Reply:
Sounds good, I tried this one is working: 
```
ALTER TABLE flows
    DROP COLUMN egressName,
    DROP COLUMN egressIP;
```
Also did a similiar change on the add column statements.